src/features/game/types/decorations.ts

```diff
@@ -389,6 +390,10 @@ export const DECORATION_DIMENSIONS: Record<DecorationName, Dimensions> = {
     width: 1,
     height: 1,
   },
+  "Volcano Gnome": {
```

Comment:
The gnome is popping up in the decorations section, I think it is missing a buff label.

<img width="144" alt="Screenshot 2025-01-22 at 10 04 15 AM" src="https://github.com/user-attachments/assets/b6dffccc-ebac-42a3-af2a-2cce6ffc6e84" />